spec/support/shared/examples/msf/ui/console/command_dispatcher/session.rb

```diff
@@ -66,12 +66,12 @@
           allow(session).to receive(:next_session=)
         end
 
-        let(:new_session_id) { 2 }
+        let(:new_session_id) { '2' }
 
         it 'backgrounds the session and switches to the new session' do
-          subject.cmd_sessions(new_session_id)
+          subject.cmd_sessions('-i', new_session_id)
```

Comment:
Is there a reason why this test was changed, rather than a new one added for `sessions -i <id>`? It seems like now we only test the `-i` functionality, with the previous functionality of `sessions <id>` not being tested.

Reply:
Initially I thought since both use the same back-end, a single test would suffice. But now in hindsight, I feel we'd need two separate tests. Will add it👍